src/databricks/labs/ucx/workspace_access/secrets.py

```diff
@@ -10,22 +10,26 @@
 from databricks.sdk.service import workspace
 from databricks.sdk.service.workspace import AclItem
 
-from databricks.labs.ucx.workspace_access.base import AclSupport, Permissions
+from databricks.labs.ucx.workspace_access.base import AclSupport, Permissions, StaticListing
 from databricks.labs.ucx.workspace_access.groups import MigrationState
 
 logger = logging.getLogger(__name__)
 
 
 class SecretScopesSupport(AclSupport):
-    def __init__(self, ws: WorkspaceClient, verify_timeout: timedelta | None = None):
+    def __init__(
+        self,
+        ws: WorkspaceClient,
+        verify_timeout: timedelta | None = None,
+        include_object_permissions: list[str] | None = None,
```

Comment:
Please add here that this parameter is intended for testing only

Reply:
this is used by other ACL support classes as well - added comments for all of them